src/main/java/net/id/incubus_core/mixin/woodtypefactory/sign/BlockEntityTypeMixin.java

```diff
@@ -20,8 +21,14 @@ public class BlockEntityTypeMixin {
     @Final
     private Set<Block> blocks;
 
+    /**
+     * Makes the set of blocks in the BlockEntityTypes mutable, so that they can be modified.
+     * Only do this once the set has been created (made immutable).
```

Comment:
This comment implies that there are circumstances where this code doesn't run. Doesn't that mean the set won't be made immutable?

Reply:
This code always runs, as it happens in the constructor